docs/proposals/20241127-predownload-in-vk-zh.md

```diff
@@ -0,0 +1,144 @@
+---
+title: 镜像预热支持预热vk
+authors:
+  - "@Abner-1"
+reviewers:
+  - "@YYY"
+creation-date: 2024-11-27
+last-updated: 2024-11-27
+status: experimental
+see-also:
+  - "/docs/proposals/20230323-imagelistpulljob.md"
+---
+
+# 镜像预热支持预热vk
+
+## Table of Contents
+* [镜像预热支持预热vk](#镜像预热支持预热vk)
+   * [Table of Contents](#table-of-contents)
+   * [Summary](#summary)
+   * [Motivation](#motivation)
+      * [Goals](#goals)
+      * [Non-Goals/Future Work](#non-goalsfuture-work)
+   * [Proposal](#proposal)
+      * [User Stories](#user-stories)
+         * [Serverless 场景](#serverless-场景)
+         * [混合部署弹性场景](#混合部署弹性场景)
+      * [Implementation Details/Notes/Constraints](#implementation-detailsnotesconstraints)
+         * [如何指定调度到 vk](#如何指定调度到-vk)
+         * [指定预热镜像后，如何进行预热](#指定预热镜像后如何进行预热)
+         * [如何通过预热已有的 pod 来实现弹性池的预热呢？](#如何通过预热已有的-pod-来实现弹性池的预热呢)
+   * [Additional Details](#additional-details)
+      * [Test Plan [optional]](#test-plan-optional)
+   * [Implementation History](#implementation-history)
+
+## Summary
+
+镜像拉取作为加速pod启动的重要一环，我们之前已经支持了基于 kruise-daemon 调用 CRI 接口进行镜像预热的方案。但这种方案无法实现 virtual kubelet 场景下的镜像预热。
+我们注意到如 ACK 平台上 ECI 上已经支持了部分预热的方案，即对预热镜像的容器数据盘进行打快照来实现镜像加速。该提案旨在利用该机制实现 vk 场景下的镜像预热，并尽可能复用已有的 CR。
+
+## Motivation
+
+弹性节点或者 serverless 作为使用云的最大优势之一，其最大的优势在于弹性，如果能够将预热镜像的机制与弹性池的机制结合起来，将能够显著提升弹性池的弹性。
+
+### Goals
+
+- 尽可能复用已有 CR 的情况下支持预热vk
+- 支持用户自行指定预热镜像
+- 支持从已有pod上进行预热
+
+### Non-Goals/Future Work
+
+- 如果 vk本身不支持预热，本提案不负责实现
+
+## Proposal
+对于用户在 ImageListPullJob 指定的镜像，我们根据指定的镜像创建一个临时pod，并从该pod上进行预热。
+
+1. 增加一个字段标识是否需要在 vk 上进行预热
+计划在已有的 ImageListPullJob CR 上增加一个字段 `spec.predownloadInVK`，用于标识是否需要在 vk 上进行预热
+
+2. 增加一个字段标识预热 vk 的selector
+计划在 ImageListPullJob 中增加一个字段 `spec.vkConfig.selector`，用于标识预热 vk 的 selector, 该字段为可选字段，如果不指定则默认使用 `type = virtual-kubelet` 的 selector.
```

Comment:
it is not necessary to add new field, the default selector should be OK in most cases

Reply:
Maybe we can allow user specify it in default config.